WalletWasabi.Gui/Shell/MainMenu/HelpMainMenuItems.cs

```diff
@@ -57,33 +57,38 @@ public HelpMainMenuItems(IMenuItemFactory menuItemFactory)
 
 #endif
 
-		[ExportMainMenuItem("Help", "Customer Support")]
+		[ExportMainMenuItem("Help", "Status Page")]
 		[DefaultOrder(2)]
 		[DefaultGroup("Support")]
+		public IMenuItem StatusPage => MenuItemFactory.CreateCommandMenuItem("Help.StatusPage");
+
+		[ExportMainMenuItem("Help", "Customer Support")]
+		[DefaultOrder(3)]
+		[DefaultGroup("Support")]
```

Comment:
Why is it renamed?

Reply:
It's not renamed, it's a new entry above the "Customer Support". See https://github.com/zkSNACKs/WalletWasabi/blob/d8bb82e7c4e8cda3ffb935f131b894369685be9d/WalletWasabi.Gui/Shell/MainMenu/HelpMainMenuItems.cs

Often a Status Page removes the load on the Customer Support, that's why the Status Page should be listet before customer support IMHO.